fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/ToolbarMenu.kt

```diff
@@ -18,6 +18,11 @@ interface ToolbarMenu {
          */
         object OpenInRegularTab : Item()
         object FindInPage : Item()
+
+        /**
+         * Opens the translations flow.
+         */
+        object Translations : Item()
```

Comment:
I have a slight preference for `Translate` or `TranslatePage` here, based on the other usage.

```suggestion
        object Translate : Item()
```

Reply:
Done